client-apis/metrics-api/src/main/java/org/wildfly/swarm/microprofile/metrics/cdi/AnnotatedDecorator.java

```diff
@@ -1,19 +1,21 @@
-/**
- * Copyright © 2013 Antonin Stefanutti (antonin.stefanutti@gmail.com)
- * <p>
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
- * <p>
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+/*
```

Comment:
We can't remove Antonin Stefanutti's copyright, as that's where the code was taken from by @pilhuhn

Reply:
ok